### ISSUE TITLE
No module named 'github_traffic_stats2'

### ISSUE THREAD

**Issue by jahnf** [seladb/github-traffic-stats#5]
Hello, thank you very much for this python tool.

I installed successfully with `pip3 install github_traffic_stats2` (Ubuntu 18.04),
but when running `github_traffic_stats` I get the following error.
```
$ github_traffic_stats 
Traceback (most recent call last):
  File "/usr/local/bin/github_traffic_stats", line 7, in <module>
    from github_traffic_stats2.github_traffic_stats import main
ModuleNotFoundError: No module named 'github_traffic_stats2'
```

Edit: With cloning and running the script directly I don't get this error. Seems to be an issue with the installation via pip.

**Comment by seladb** [comment]
let me check and get back to you

**Comment by seladb** [comment]
@jahnf Fixed in version [20.6.6](https://pypi.org/project/github-traffic-stats2/). Please try again and let it know if it works

**Comment by jahnf** [comment]
Yes thanks. Works for me with now with pip.
